test/jdk/java/nio/channels/Selector/RacyDeregister.java

```diff
@@ -39,6 +39,12 @@
  * @author Frank Ding
  * @run main/timeout=1200 RacyDeregister
  */
+
+/* @test
+ * @requires (os.family == "windows")
+ * @run main/othervm/timeout=1200 -Djava.nio.channels.spi.SelectorProvider=sun.nio.ch.WindowsSelectorProvider RacyDeregister
+ */
+
```

Comment:
I like how this test can be run with the system property set for Windows only - nice!